src/ripple/overlay/impl/PeerImp.cpp

```diff
@@ -896,6 +909,16 @@ PeerImp::doProtocolStart()
 // Called repeatedly with protocol message data
 void
 PeerImp::onReadMessage(error_code ec, std::size_t bytes_transferred)
+{
+    this->onReadMessageInternal(ec, bytes_transferred);
```

Comment:
The `this->` is almost certainly redundant. It's also unclear why this needs to be a separate function?

Is the intention to _always_ prevent an error message if `ec` is set? I'm not sure I think this level of logging is needed, buf if that's the intention why not simply define `onReadMessageInternal` as a directly-invoked lambda?

Reply:
Done, removed.
(This was a remnant of debugging when I wanted to be sure there weren't any unreported errors. The only reason for the "Internal" method was to be able to log any error codes.)